tests/absa/test_training_protocol.py

```
@@ -0,0 +1,227 @@
+    ]
+    test = [
+        _row("test-1", "excellent food", "food", "positive", "test"),
+        _row("test-2", "poor service", "service", "negative", "test"),
+        _row("test-3", "standard menu", "menu", "neutral", "test"),
+    ]
+    return train, test
+
+
+def test_seed_everything_reproduces_python_numpy_torch_and_loader_generator() -> None:
+    first_generator = seed_everything(17)
+    first = (random.random(), np.random.random(), torch.rand(1), torch.rand(1, generator=first_generator))
```

Ruff (0.15.21)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)